packages/daimo-api/src/server/tokenRegistry.ts

```diff
@@ -0,0 +1,94 @@
+import { ForeignCoin, USDbC, assert, getHomeCoinToken } from "@daimo/common";
+import { Address, getAddress, isAddress } from "viem";
+
+import { chainConfig } from "../env";
+import { fetchWithBackoff } from "../network/fetchWithBackoff";
+
+// Exclude native ETH
+export type ForeignToken = ForeignCoin & { token: Address };
```

Comment:
`ForeignCoin` vs `ForeignToken` is going to be very easy to mix up. can we combine?

Reply:
Yes sure -- this is what we had before, I think because "ETH" was separated out (token is "ETH" instead of the WETH address)

wasn't sure if there was a reason for that so left it in. but i can refactor